logging/interceptor.go

```diff
@@ -161,7 +217,7 @@ func addRequestIDField(ctx context.Context, fields logrus.Fields) (context.Conte
 
 	fields[DefaultRequestIDKey] = reqID
 
-	return metadata.AppendToOutgoingContext(ctx, DefaultRequestIDKey, reqID), nil
+	return ctx, nil
```

Comment:
If we are not changing anything in context, can we remove it from return statement ?